subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -842,11 +842,14 @@ uint8_t ll_rx_get(void **node_rx, uint16_t *handle)
 							  mfifo_tx_ack.l);
 			} while ((cmplt_prev != 0U) ||
 				 (cmplt_prev != cmplt_curr));
+#endif /* CONFIG_BT_CONN */
 
+			if (0) {
+#if defined(CONFIG_BT_CONN)
```

Comment:
@ppryga let me know the Kconfig you want to OR here for DF related feature, I will add  it.

Reply:
@cvinayak This is DF independent. This is connected with periodic advertising only. If the DF is disabled and periodic adv. scanning is enabled this code the may be node_rx->type == NODE_RX_TYPE_RELEASE.
So I propose to use CONFIG_BT_CTLR_SYNC_PERIODIC or CONFIG_BT_PER_ADV_SYNC.